src/gsi/radinfo.f90

```diff
@@ -1135,7 +1135,7 @@ subroutine radinfo_read
                 nusis(j),nuchan(j),' not found in satbias_in file - set to zero '
              endif
           end do
-140       format(i4,1x,a20,12f12.6)
+140       format(i5,1x,a20,12f12.6)
```

Comment:
Seems we use `i6` elsewhere for `nusis`  Should `i5` here be `i6`?

Reply:
@RussTreadon-NOAA I made the other reads/writes consistent with this one.  No changes were made.